projects/angular/src/popover/tooltip/tooltip-content.ts

```diff
@@ -122,6 +122,11 @@ export class ClrTooltipContent extends AbstractPopover {
     this.updateCssClass({ oldClass: `tooltip-${oldSize}`, newClass: `tooltip-${newSize}` });
   }
 
+  ngOnInit() {
+    this.size = this.size || defaultSize;
```

Comment:
Just for my own curiosity, what's so different about this vs the other way the default was getting set (or trying to) before?